const server = require('./app')

// process.on('unhandledRejection', (err) => {
//   console.log('UNHANDLED REJECTION! ðŸ’¥ Shutting down...')
//   console.log(err.name, err.message)
//   server.close(() => {
//     process.exit(1)
//   })
// })

// process.on('SIGTERM', () => {
//   console.log('ðŸ‘‹ SIGTERM RECEIVED. Shutting down gracefully')
//   server.close(() => {
//     console.log('ðŸ’¥ Process terminated!')
//   })
// })

// process.on('uncaughtException', (err) => {
//   console.log('UNCAUGHT EXCEPTION! ðŸ’¥ Shutting down...')
//   console.log(err)
//   process.exit(1)
// })